### PR TITLE
Require PHP 5.6 upgrade components.

### PR DESCRIPTION
**Pull request by relaxnow**
Fixes #101

### CODE REVIEW

**Comment by rjkip** [comment]
Nice decision documentation @ 67f79c0 :+1:

If you verified authentication still works properly, :shipit:

**Comment by rjkip** [comment]
Looking at the build status, please verify the application and authentication still works. Also, I'd suggest removing the PHP 5.4 and 5.5 builds.

**Comment by relaxnow** [comment]
Actually I'd been running the saml-bundle dev-develop for some time and wondered why I had to modify the Provider 😅 . So I'm fairly sure it works, also I've just tested it again and reviewd the upgrade notes.

File: src/Surfnet/StepupRa/RaBundle/Service/VettingService.php [hunk 1]
```diff
@@ -46,6 +46,7 @@
 /**
  * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
  * @SuppressWarnings(PHPMD.TooManyMethods)
```
Comment (rjkip) [inline src/Surfnet/StepupRa/RaBundle/Service/VettingService.php]: Can this be removed? Otherwise, if authentication indeed works, :+1:.

Comment (relaxnow) [inline src/Surfnet/StepupRa/RaBundle/Service/VettingService.php]: It appears so. Removed.

**Pull request merged**